composer.json

```
@@ -93,9 +93,10 @@
         "process-timeout": 3000
     },
     "scripts": {
-        "fix-cs": "php-cs-fixer fix -v --show-progress=estimating",
-        "unit": "paratest -p half -c phpunit.xml --runner=WrapperRunner",
-        "integration": "paratest -p half -c phpunit-integration-legacy.xml --runner=WrapperRunner",
+        "fix-cs": "php-cs-fixer fix -v --show-progress=dots",
+        "check-cs": "@fix-cs --diff --config=.php_cs --dry-run",
```

Configuration file spec is either redundant (since it's the default) or might lead to differences between `fix-cs` and `check-cs`.

```suggestion
        "fix-cs": "php-cs-fixer fix -v --config=.php_cs --show-progress=dots",
        "check-cs": "@fix-cs --diff --dry-run",
```